pkg/aspect/accessLogsManager_test.go

```diff
@@ -54,36 +74,20 @@ func TestAccessLoggerManager_NewAspect(t *testing.T) {
 		aspect: tl,
 	}
 
-	// TODO: add back tests for custom when we introduce descriptors
-	//customExec := &accessLogsExecutor{
-	//	name:   "custom_access_log",
-	//	aspect: tl,
-	//	labels: map[string]string{"test": "test"},
-	//}
-
 	combinedStruct := &aconfig.AccessLogsParams{
 		LogName: "combined_access_log",
 		Log: &aconfig.AccessLogsParams_AccessLog{
-			LogFormat: aconfig.COMBINED,
+			DescriptorName: "common",
```

Comment:
I was chatting with Doug about this (and more generally how we've been going round and round on this) and we came to the same conclusion: we'll have to figure out some way to package an always-available istio-global-config with things like these descriptors, the common metric descriptors, and so on. I'm actually not opposed to it being static protos that we compile into the binary (speaking of going round-and-round).